ynr/apps/elections/api/next/api_views.py

```diff
@@ -0,0 +1,164 @@
+import json
+
+from django.http import HttpResponse
+from django.db.models import Prefetch
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import cache_page
+from django.views import View
+from rest_framework.decorators import action
+from rest_framework import viewsets
+from rest_framework.response import Response
+from six import text_type
+
+import elections.api.next.serializers
+from api.next.views import ResultsSetPagination
+from candidates import models as extra_models
+from candidates.api.next.serializers import LoggedActionSerializer
+from elections.models import Election
+from elections.uk.geo_helpers import (
+    get_ballots_from_coords,
+    get_ballots_from_postcode,
+)
+from elections.filters import BallotFilter
+from official_documents.models import OfficialDocument
+from popolo.models import Membership
+from utils.db import LastWord
+
+
+class UpcomingElectionsView(View):
```

Comment:
Is this view actually mounted on any route? There's a separate version of this view in the `v0.9` codebase to cover the `v0.9` endpoint. If its unreachable code, we can just bin it.